README.MD

```diff
@@ -35,7 +36,30 @@ module: {
     loaders: [
         {
             test: /\.js$/,
-            loader: WebPackAngularTranslate.jsLoader()
+            loader: WebPackAngularTranslate.jsLoader(),
+            query: {
+                sourceType: 'script', // acorn option default
+                allowImportExportEverywhere: false // acorn option default
```

Comment:
I don't think that this option is needed in all cases. Normally the `sourceType` option should be sufficient. The latter is used in cases where imports are used inside of functions, and therefore, is not conform with the current spec (I believe)